src/app_service/service/chat/dto/chat.nim

```diff
@@ -252,10 +252,12 @@ proc toChannelGroupDto*(jsonObj: JsonNode): ChannelGroupDto =
   discard jsonObj.getProp("muted", result.muted)
 
 # To parse Community chats to ChatDto, we need to add the commuity ID and type
-proc toChatDto*(jsonObj: JsonNode, communityId: string): ChatDto =
+proc communityChatToChatDto*(jsonObj: JsonNode, communityId: string): ChatDto =
+  jsonObj["communityId"] = %* communityId
+  jsonObj["chatType"] = %* 6
+  # It's very important to update jsonNode properly before `toChatDto` proc call.
+  # `chatId` should be composed of `communityId` + `chatId` and used in that form across the app.
```

Comment:
It feels a bit off to me that updating the properties as done here before calling `toChatDto` becomes a requirement.
Is this data not available or is this just converting data types?

Reply:
@PascalPrecht I added a comment [here](https://github.com/status-im/status-desktop/issues/5770#issuecomment-1128780296), so there is an inconsistency using `chatId` in community responses and other responses and status-go signals' data. That `communityChatToChatDto` proc is called only for responses received from community related calls that they can match the form of `ChatDto` we're using across the app. This is an example how "chat" object returned from for a community looks (missing a lot of details we need):
```
"88652534-9557-4d31-8fc2-0778d8e1f653": {
          "id": "88652534-9557-4d31-8fc2-0778d8e1f653",
          "name": "general",
          "color": "#1FB9DF",
          "emoji": "",
          "description": "General channel for the community",
          "members": null,
          "permissions": {
            "access": 1
          },
          "canPost": true,
          "position": 0,
          "categoryID": ""
        }
```

All in all that fix is important one at this moment, and will be until we sort out all calls/responses. :)